python/Ganga/Core/GangaRepository/__init__.py

```diff
@@ -20,3 +20,10 @@ def getRegistries():
 
 def getRegistry(name):
     return allRegistries[name]
+
+def getRegistrySlice(name):
+    return allRegistries[name].getSlice()
+
+def getRegistryProxy(name):
+    return allRegistries[name].getProxy()
+
```

Comment:
Frankly they're 1 line wrappers and other pre-existing functions don't... but there's an edit button on github :)